app/styles/_shame.scss

```diff
@@ -42,7 +42,7 @@
   position: absolute;
   right: 0;
   max-width: 480px;
-  top: $au-unit-large - $au-unit-tiny/2;
+  top: $au-unit-large - calc($au-unit-tiny / 2);
```

Comment:
Does this yield the expected result? Isn't `calc` a runtime css thing? I think you can simply multiply by `0.5` instead, or use the math.div thing that the deprecation message suggests(ed?).

Reply:
Also, I think these styles are simply out-of-date in this project. The `.alert` class isn't used anymore and it currently looks like this:

![image](https://user-images.githubusercontent.com/3533236/175980051-d099ad87-8448-4e3e-ab05-7292c0fa0547.png)

So I think we can give improve that in some future PRs 😄. I guess the Loket app has a modernized setup.

Reply:
I can't remember, I will run the app tomorrow to see if the result is correct. Did not knew about math.div